src/overlays/actors/ovl_Bg_Gate_Shutter/z_bg_gate_shutter.c

```diff
@@ -45,9 +45,7 @@ void BgGateShutter_Init(Actor* thisx, GlobalContext* globalCtx) {
     DynaPolyInfo_SetActorMove(&this->dyna, 0);
     DynaPolyInfo_Alloc(&D_06001DA8, &local_c);
     this->dyna.dynaPolyId = DynaPolyInfo_RegisterActor(globalCtx, &globalCtx->colCtx.dyna, thisx, local_c);
-    this->somePosX = thisx->posRot.pos.x;
-    this->somePosY = thisx->posRot.pos.y;
-    this->somePosZ = thisx->posRot.pos.z;
+    VEC_SET(this->somePos, thisx->posRot.pos.x, thisx->posRot.pos.y, thisx->posRot.pos.z);
```

Comment:
Setting each float individually is necessary, as it is not a struct copy, which is why I thought using the macro would be cleaner. I can do this->somePos.x = thisx->posRot.pos.x for xyz if you'd prefer that, but I thought the macro would be better since we have it

Reply:
The macro is meant mostly for cases where it's needed due to same line compiler memes. Otherwise it should generally be avoided for consistency and simplicity.

Reply:
Oh i saw your comment about VEC_SET from the other time I used it. The same probably applies here so ill change it